Makefile

```diff
@@ -0,0 +1,38 @@
+ROOT_DIR := $(patsubst %/,%,$(dir $(abspath $(lastword $(MAKEFILE_LIST)))))
+BUILD_DIR := $(ROOT_DIR)/.build
+OUT_DIR := $(ROOT_DIR)/src/proto
+
+KAVA_TAG ?= upgrade-v44
```

Comment:
I ran with `KAVA_TAG=nd-add-all-third-party-protos make all` since some additions files are not in the upgrade branch or a tagged version.